src/components/DarkModeToggle/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react'
+import { useValues, useActions } from 'kea'
+import { layoutLogic } from '../../logic/layoutLogic'
+import './style.scss'
+
+export const DarkModeToggle = () => {
+    const { websiteTheme } = useValues(layoutLogic)
+    const { setWebsiteTheme } = useActions(layoutLogic)
+
+    useEffect(() => {
+        setWebsiteTheme(window.__theme)
+        window.__onThemeChange = () => setWebsiteTheme(window.__theme)
```

Comment:
Unsure whether it's best practice to call a kea action from a window event, but maybe @mariusandra has better context on this?

Reply:
Happy to take more comments here but we did discuss this dark mode implementation before.

This has also just moved from somewhere else, rather than been added in this PR.